aiohttp/web.py

```diff
@@ -349,7 +349,7 @@ def main(argv):
         type=int,
         default="8080"
     )
-    args, extra_args = arg_parser.parse_known_args(argv)
+    args, extra_args = arg_parser.parse_known_args(argv[1:])
```

Comment:
Please change. I have a guts feeling that accepting `sys.argv[1:]` is better.
Also I saw the similar approach in past a veral times.